api/spec/services/ingestor/strategy/epub/creator/text_titles_spec.rb

```diff
@@ -30,12 +29,12 @@
   end
 
   it "updates existing objects rather than create new ones" do
-    text = Text.new(:unique_identifier => '1234')
+    text = Text.new(unique_identifier: "1234")
     @title_nodes.each do |node|
       node_inspector = MetadataInspector.new(node, @metadata_node)
-      text.titles.build(:value => node_inspector.text.presence)
+      text.titles.build(value: node_inspector.text.presence)
     end
-    expect(@titles_creator.create(@title_nodes, text.titles).first).to eq(text.titles.first)
+    expect(@titles_creator.create(@title_nodes, text.titles).first)
+      .to eq(text.titles.first)
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.